src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -408,4 +408,15 @@ public ScmPolling pollScm() {
 
         return links;
     }
+
+    /**
+     * Deletes the current job
+     */
+    public void delete() {
+        this.open();
+        clickLink("Delete Project");
+        // We need to wait a little for the alert to appear
+        elasticSleep(500);
```

Comment:
Given that alert [throws](http://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/WebDriver.TargetLocator.html#alert--) an exception if there's no alert, could we go for a "wait until" (with timeout) kind of waiting.